guides/common/modules/ref_ports-and-firewall-requirements.adoc

```diff
@@ -32,16 +32,17 @@ The following tables indicate the destination port and the direction of network
 | 69 | UDP | TFTP | Client | TFTP Server (optional) |
 | 443 | TCP | HTTPS | {SmartProxy} | {ProjectName} API | Communication from {SmartProxy}
 ifdef::katello,satellite,orcharhino[]
-| 443, 80 | TCP | HTTPS, HTTP | Client | Content Retrieval | Content
-| 443, 80 | TCP | HTTPS, HTTP | {SmartProxy} | Content Retrieval | Content
-| 443, 80 | TCP | HTTPS, HTTP| Client | Content Host Registration | {SmartProxy} CA RPM installation
 | 443 | TCP | HTTPS | Client | Content Host registration | Initiation
 
 Uploading facts
 
 Sending installed packages and traces
```

Comment:
Let's remove this, because we don't know for sure for which registration method it's supposed to be (at least I don't know)

Reply:
These processes take place regardless of any method and via `443` only. Can we replace it with simple `Host Registration` to avoid confusion?